js/src/components/google-account-card/authorize-google-account-card.js

```diff
@@ -7,116 +7,60 @@ import { createInterpolateElement } from '@wordpress/element';
-	disabled = false,
-} ) {
-	const pageName = glaData.mcSetupComplete ? 'reconnect' : 'setup-mc';
-	const isAskingScope = Boolean( additionalScopeEmail );
-	const { createNotice } = useDispatchCoreNotices();
-	const [ fetchGoogleConnect, { loading, data } ] = useGoogleAuthorization(
-		pageName,
+export default function AuthorizeGoogleAccountCard( { additionalScopeEmail } ) {
+	const [ handleConnect, { loading, data } ] = useGoogleConnectFlow(
 		additionalScopeEmail
 	);
 
-	const handleConnectClick = async () => {
-		try {
-			const { url } = await fetchGoogleConnect();
-			window.location.href = url;
-		} catch ( error ) {
-			createNotice(
-				'error',
-				__(
-					'Unable to connect your Google account. Please try again later.',
-					'google-listings-and-ads'
-				)
-			);
-		}
-	};
-
-	const description = isAskingScope ? (
-		<>
-			{ additionalScopeEmail }
-			<p>
-				<em>
-					{ createInterpolateElement(
-						__(
-							'<alert>Error:</alert> You did not allow WooCommerce sufficient access to your Google account. You must select all checkboxes in the Google account modal to proceed. <link>Read more</link>',
-							'google-listings-and-ads'
-						),
-						{
-							alert: (
-								<span className="gla-authorize-google-account-card__error-text" />
-							),
-							link: readMoreLink,
-						}
-					) }
-				</em>
-			</p>
-		</>
-	) : (
-		<>
-			{ __(
-				'Required to sync with Google Merchant Center and Google Ads.',
-				'google-listings-and-ads'
-			) }
-			<p>
-				<em>
-					{ createInterpolateElement(
-						__(
-							'You will be prompted to give WooCommerce access to your Google account. Please check all the checkboxes to give WooCommerce all required permissions. <link>Read more</link>',
-							'google-listings-and-ads'
-						),
-						{ link: readMoreLink }
-					) }
-				</em>
-			</p>
-		</>
-	);
-
-	const buttonText = isAskingScope
-		? __( 'Allow full access', 'google-listings-and-ads' )
-		: __( 'Connect', 'google-listings-and-ads' );
-
 	return (
 		<AccountCard
 			appearance={ APPEARANCE.GOOGLE }
-			disabled={ disabled }
 			alignIcon="top"
-			description={ description }
+			description={
+				<>
+					{ additionalScopeEmail }
+					<p>
+						<em>
+							{ createInterpolateElement(
+								__(
+									'<alert>Error:</alert> You did not allow WooCommerce sufficient access to your Google account. You must select all checkboxes in the Google account modal to proceed. <link>Read more</link>',
+									'google-listings-and-ads'
+								),
+								{
+									alert: (
+										<span className="gla-authorize-google-account-card__error-text" />
+									),
+									link: readMoreLink,
+								}
+							) }
+						</em>
+					</p>
+				</>
+			}
 			alignIndicator="top"
 			indicator={
 				<AppButton
 					isSecondary
-					isDestructive={ isAskingScope }
-					disabled={ disabled }
+					isDestructive
 					loading={ loading || data }
 					eventName="gla_google_account_connect_button_click"
```

Comment:
Similar to https://github.com/woocommerce/google-listings-and-ads/pull/1074#discussion_r741671498. But I'm thinking maybe add `eventProps` here to force them to appear in conflict, so that we would not miss it. 😆

Reply:
Done in [`65478e6` (#1074)](https://github.com/woocommerce/google-listings-and-ads/pull/1074/commits/65478e6823aa646e98ac46902759d085100671a4).